posthog/hogql/constants.py

```diff
@@ -110,7 +112,11 @@
     "distinct_id",
     "elements_chain",
     "created_at",
+    "person_id",
+    "person.created_at",
+    "person.properties",
 ]
 
 # Never return more rows than this in top level HogQL SELECT statements
+DEFAULT_RETURNED_ROWS = 100
```

Comment:
This used to be 1000, but went a bit down to keep the output manageable in the data tables if you accidentally forget to add a limit yourself.